components/prism-java.js

```diff
@@ -55,7 +55,9 @@
 		'operator': {
 			pattern: /(^|[^.])(?:<<=?|>>>?=?|->|--|\+\+|&&|\|\||::|[?:~]|[-+*/%&|^!=<>]=?)/m,
 			lookbehind: true
-		}
+		},
+		'constant': /\b[A-Z][A-Z_\d]{2,}\b/,
+		'variable': /\b[a-z]\w*\b/
```

Comment:
I think the `variable` pattern has too many false positives to include. Class fields (instance and static) commonly use camelCase, so this regex will misidentify all of them as variables. This is just how Java's syntax is, so I don't think that we can fix this. Could you please remove the `variable` pattern?

Reply:
done